docs/admin/installation/installation_overview.rst

```diff
@@ -1,6 +1,29 @@
 Installation Overview
 =====================
 
+Before You Begin
```

Comment:
I like this placement, but note that we have a "Before You Begin" section later in the installation guide as well, which also speaks to support from FPF.

I would suggest re-titling this section "Installation support" or similar, and consolidating all this information here.